src/plothist/histogramming.py

```diff
@@ -51,7 +51,7 @@ def create_axis(bins, range=None, data=np.array([])):
                 "Please supply a range or provide data."
             )
         x_min = min(data) if range[0] == "min" else range[0]
-        x_max = max(data) if range[1] == "max" else range[1]
+        x_max = max(data)+((max(data)-min(data))/(bins-1)) if range[1] == "max" else range[1]
```

Comment:
Need to consider the cases bins<=0 and bins=1

Reply:
Here, what I do not like, it that we do not apply what the user asks us to do. The user want the upper range to be max(data) and we do not do that. We should mention it explicitly in the documentation of the function and also in the readthedocs. I propose you put the example you show in the summary of this PR in the readthedoc to explain the "exclusive upper-edge" situation.

Reply:
"you put the example you show in the summary of this PR in the readthedoc to explain the "exclusive upper-edge" situation."
In which section? I think the docstring is enough.